### PR TITLE
Add tabbar sample

### PR DESCRIPTION
**Pull request by kazo0**
#310 

## PR Type

What kind of change does this PR introduce?

- Feature

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested on iOS.
- [ ] Tested on Wasm.
- [ ] Tested on Android.
- [ ] Tested on UWP.
- [ ] Tested in both **Light** and **Dark** themes.
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.Shared/Entities/SampleCategory.cs [hunk 1]
```diff
@@ -15,6 +15,7 @@ public enum SampleCategory
 		Theme,
 		Components,
 		Features,
+		Toolkit,
```
Comment (agneszitte-nventive) [inline Uno.Gallery/Uno.Gallery.Shared/Entities/SampleCategory.cs:18]: Can we have two sections under Components ? 
> Components
    > WinUI controls
    > Toolkit controls

File: Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+      mc:Ignorable="d">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <utu:NavigationBar toolkit:VisibleBoundsPadding.PaddingMask="Top"
```
Comment (agneszitte-nventive) [inline Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml:18]: Line-space can be added here after RowDefinitions

File: Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                    <StackPanel HorizontalAlignment="Center"
+                                VerticalAlignment="Center">
+                        <FontIcon Glyph="&#xE946;" />
+                        <TextBlock Text="About"
+                                   TextAlignment="Center" />
+                    </StackPanel>
+                </Border>
+            </Grid>
+        </Grid>
+
```
Comment (agneszitte-nventive) [inline Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml:77]: Double-space line can be removed here

File: Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/MaterialBottomBarSampleNestedPage.xaml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+      mc:Ignorable="d">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <utu:NavigationBar Content="Material Bottom TabBar Sample Page"
```
Comment (agneszitte-nventive) [inline Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/MaterialBottomBarSampleNestedPage.xaml:18]: Line-space can be added here after RowDefinitions

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/TabBarSamplePage.xaml [hunk 5]
```diff
@@ -0,0 +1,261 @@
+﻿<Page
+    x:Class="Uno.Gallery.Views.SamplePages.TabBarSamplePage"
```
Comment (agneszitte-nventive) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/TabBarSamplePage.xaml:2]: Format XAML page please ;)
image
...

File: Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/TabBarSamplePage.xaml [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+    xmlns:utu="using:Uno.Toolkit.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <sample:SamplePageLayout>
+        <sample:SamplePageLayout.MaterialTemplate>
+            <DataTemplate>
+                <StackPanel Padding="0,20"
```
Comment (agneszitte-nventive) [inline Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/TabBarSamplePage.xaml:15]: Please use XamlDisplay in Gallery for the samples

File: Uno.Gallery/Uno.Gallery.Shared/ViewModels/ViewModelBase.cs [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+				_propertyValueStore[propertyName] = value;
+				PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+			}
+		}
+#if IS_WINUI
+		[Microsoft.UI.Xaml.Data.Bindable]
+#else
+		[Windows.UI.Xaml.Data.Bindable]
+#endif
+		public class Command : ICommand
```
Comment (Xiaoy312) [inline Uno.Gallery/Uno.Gallery.Shared/ViewModels/ViewModelBase.cs:36]: move this class into its own file, and consider exposing a GetCommand method from this VMB, like done here:
https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/main/Uno/NugetPackageExplorer.Legacy/Framework/MVVM/ViewModelBase.cs#L81-L94

**Review by Xiaoy312 — APPROVED** [review]

**Pull request merged**